saleor/graphql/schema.graphql

```diff
@@ -1831,11 +1831,75 @@ type Image {
   alt: String
 }
 
+type Invoice implements Node & ObjectWithMetadata {
+  id: ID!
+  created: DateTime!
+  number: String!
+  order: Order!
+  url: String
+  privateMetadata: [MetadataItem]!
+  metadata: [MetadataItem]!
+  privateMeta: [MetaStore]! @deprecated(reason: "DEPRECATED: Will be removed in Saleor 2.11. use the `privetaMetadata` field instead. ")
+  meta: [MetaStore]! @deprecated(reason: "DEPRECATED: Will be removed in Saleor 2.11. use the `metadata` field instead. ")
+}
+
+type InvoiceCreate {
+  errors: [Error!]!
+  invoice: Invoice
+  invoiceErrors: [InvoiceError!]!
+  order: Order
+}
+
+type InvoiceDelete {
+  errors: [Error!]!
+  invoiceErrors: [InvoiceError!]!
+  order: Order
+}
+
+type InvoiceError {
+  code: InvoiceErrorCode!
+  field: String
+  message: String
+}
+
+enum InvoiceErrorCode {
+  PLUGIN_NOT_ENABLED
+}
+
+type InvoiceUpdate {
+  errors: [Error!]!
+  invoice: Invoice
+  invoiceErrors: [InvoiceError!]!
+  order: Order
+}
+
+input InvoiceInput {
+  created: DateTime
+  number: String
+  url: String
+}
+
+type InvoiceSend {
+  errors: [Error!]!
+  invoiceErrors: [InvoiceError!]!
+}
+
 input IntRangeInput {
   gte: Int
   lte: Int
 }
 
+interface Job {
+  status: JobStatus
+}
+
+enum JobStatus {
+  PENDING_CREATION
+  OK
+  PENDING_DELETION
+  DELETED
```

Comment:
For me, `SUCCESS ` sounds better than `OK`.  Also, we need status for failed, e.g. `FAILED`.